tests/cypress/scripts/cliHelper.js

```diff
@@ -7,6 +7,11 @@ import { squad } from '../config'
 
 export const cliHelper = {
   getTargetManagedCluster: () => {
+    if (Cypress.env("OPTIONS_MANAGED_CLUSTER_NAME")) {
+      cy.log(`Imported cluster name found: ${Cypress.env("OPTIONS_MANAGED_CLUSTER_NAME")}`)
+      return cy.wrap(Cypress.env("OPTIONS_MANAGED_CLUSTER_NAME"))
+    }
+
```

Comment:
If the `OPTIONS_MANAGED_CLUSTER_NAME` is exported, use this managed cluster name for the test.